README.md

```diff
@@ -271,5 +271,5 @@ $ ./docker-inspire run --rm web poetry run inspirehep importer records -d data/r
 # Local
 $ poetry run inspirehep importer records -f data/records/literature/374836.json -f data/records/authors/999108.json
 # Docker
-$ ./docker-inspire run --rm web poetry run inspirehep importer records -f data/records/literature/374836.json -f data/records/authors/999108.json
+$ docker-compose run --rm web poetry run inspirehep importer records -f data/records/literature/374836.json -f data/records/authors/999108.json
```

Comment:
it looks like `web` is defined in `override`. Does this really work?

Reply:
yes: https://docs.docker.com/compose/extends/#understanding-multiple-compose-files

Reply:
in short: default `docker-compose` command is actually doing same thing as `docker-compose -f docker-compose.yml -f docker-compose.override.yml`